lib/beaker-hostgenerator/parser.rb

```diff
@@ -139,11 +139,9 @@ def tokenize_layout(layout_spec)
     def is_ostype_token?(token, bhg_version)
       BeakerHostGenerator::Data.get_platforms(bhg_version).each do |platform|
         ostype = platform.split('-')[0]
-        if ostype == token
-          return true
-        end
+        return true if ostype == token
       end
-      return false
+      false
```

Comment:
I'd go with:
```ruby
BeakerHostGenerator::Data.get_platforms(bhg_version).any? { |platform| platform.split('-')[0] == token }
```

(no GH suggestion, the diff renders horribly)

Reply:
applied it by hand